Sources/Shared/VimeoRequestSerializer.swift

```diff
@@ -156,15 +151,16 @@ final public class VimeoRequestSerializer: AFJSONRequestSerializer {
             
             if let base64String = base64String {
                 let headerValue = "Basic \(base64String)"
-                request.setValue(headerValue, forHTTPHeaderField: Constants.AuthorizationHeaderKey)
+                request.setValue(headerValue, forHTTPHeaderField: .authorizationHeaderKey)
             }
         }
         
         return request
     }
     
     private func requestModifyingUserAgentHeader(fromRequest request: URLRequest) -> URLRequest {
-        guard let frameworkVersion = Bundle(for: type(of: self)).infoDictionary?["CFBundleShortVersionString"] as? String else {
+        let infoDictionary = Bundle(for: type(of: self)).infoDictionary
+        guard let frameworkVersion = infoDictionary?["CFBundleShortVersionString"] as? String else {
```

Comment:
I still don't understand why to now Apple still hasn't made a constant available for this `CFBundleShortVersionString`. 🤦‍♂